pysph/tools/tests/test_geometry.py

```diff
@@ -17,32 +11,34 @@ class TestGeometry(unittest.TestCase):
     def test_distance(self):
         point_1 = np.random.random_sample(3)
         point_2 = np.random.random_sample(3)
-        self.assertAlmostEqual(distance(point_1, point_2),
-                               np.linalg.norm(point_1 - point_2))
+        d1 = G.distance(point_1, point_2)
+        d2 = np.linalg.norm(point_1 - point_2)
+        assert d1 == pytest.approx(d2)
 
     def test_distance_2d(self):
         point_1 = np.random.random_sample(2)
         point_2 = np.random.random_sample(2)
-        self.assertAlmostEqual(distance_2d(point_1, point_2),
-                               np.linalg.norm(point_1 - point_2))
+        d1 = G.distance_2d(point_1, point_2)
+        d2 = np.linalg.norm(point_1 - point_2)
+        assert d1 == pytest.approx(d2)
 
     def test_matrix_exp(self):
         try:
             from scipy import linalg
             e = linalg.expm
             n = np.random.randint(1, 10)
             mat = np.random.rand(n, n)
-            assert np.allclose(e(mat), matrix_exp(mat))
+            assert np.allclose(e(mat), G.matrix_exp(mat))
         except ImportError:
-            raise unittest.SkipTest()
+            raise unittest.SkipTest('Scipy is not installed')
 
     def test_extrude(self):
         n = np.random.randint(1, 20)
```

Comment:
Please do not use random numbers for the test unless absolutely necessary, otherwise it is very difficult to reproduce the test, otherwise set the random number seed.